google-cloud-core/Gemfile

```diff
@@ -4,7 +4,7 @@ gemspec
 
 gem "rake", "~> 11.0"
 gem "google-api-client", "~> 0.14.0"
```

Comment:
The google-api-client version needs to be updated to `~> 0.16.0` to match the version change made everywhere else.